examples/ee_builder_base.yaml

```diff
@@ -5,32 +5,47 @@
   collections:
     - infra.ee_utilities
   vars:
-    ee_builder_dir_clean: true
-    ah_host: ahnosso.node
-    ah_token: 4ab76668c4b2622ab1dfb7cbee7e6c9b8e62c0f5
+    # For controller configuration definition
+    ee_builder_dir_clean: false
+    builder_dir: "."
+    ee_update_base_images: false
+    ee_reg_credential: Automation Hub Container Registry
+    ee_base_image: registry.redhat.io/ansible-automation-platform-24/ee-minimal-rhel9:latest
+    ee_pull_collections_from_hub: true
+    ah_host: hub.nas
+    ah_token: ec28091dfebd9fb4c7ddc59d34cddb35350b71cb
     ee_registry_dest: ahnosso.node
     ee_registry_username: admin
     ee_registry_password: secret123
+    ee_verbosity: 3
     ee_list:
       - name: custom_ee
```

Comment:
I think it would be good to add a comment to this image build that notes the base for this image is overriding the default base image. Also, I think it would be good to add a second item that doesn't override the base.

Reply:
what would the 2nd item you had in mind be, or we have the two different one has base per list?

Reply:
I'm thinking that it would be good to show creating two different EE's. One from the base image and one with an overrided image. Maybe make the one that is in the example use the base EE instead of overriding it and add one that overrides the base EE to use the "supported" version of the EE and adds something like the community.aws collection.